code/Makefile

```diff
@@ -97,14 +115,27 @@ $(filter %$(TEST_E_SUFFIX), $(TEST_EXAMPLES)): %$(TEST_E_SUFFIX): %$(GEN_E_SUFFI
 	@mkdir -p $(LOG_FOLDER)
 	- $(DIFF) "stable/$*/" $(BUILD_FOLDER)$(EDIR)/ > $(LOG_FOLDER)$(EDIR)$(LOG_SUFFIX) 2>&1
 
+# actually run the tests
 test: $(GEN_EXAMPLES) $(TEST_EXAMPLES)
 	@echo ----------------------------
 	@echo Make complete, checking logs
 	@echo ----------------------------
 	@LOG_FOLDER=$(LOG_FOLDER) LOG_SUFFIX=$(LOG_SUFFIX) NOISY=$(NOISY) "$(SHELL)" $(SCRIPT_FOLDER)log_check.sh
 
+# generate pretty dot graphs too
+graphmod: check_stack
```

Comment:
`graphmod` target missing from `.PHONY`.

Reply:
fixed

Reply:
Looks like `graphmod` is missing in `.PHONY` with the most recent commit.

Reply:
Enjoying this, aren't you? Fixed.